src/easyscore.js

```diff
@@ -271,12 +292,18 @@ export class EasyScore {
     this.options = Object.assign({
       factory: null,
       builder: null,
+      commitHooks: [
+        setId,
+        setClass,
+        Articulation.easyScoreHook,
```

Comment:
>new elements don't ever have to touch this file.

Is that so problematic? It seems natural that the default EasyScore config is tightly coupled to the EasyScore class. Won't that make it easier to follow the "evolution" of the EasyScore defaults, ie: via git blame? Moving the registration of default hooks into their respective files unnecessarily fragments something that I think belongs together. And if the defaults ever end up changing drastically, then those changes don't permeate the codebase, but stay in an isolated place.